### DISCUSSION TITLE
Missing Consist RS Activity

### DISCUSSION THREAD

**Discussion by IsambardKingdomBrunel** [DocMartin7644/Chiltern-Route-v2#4] [General]
Hi boys,

i have a missing consist for the Milk To Marylebone activity.

Chiltern was downloaded and installed using GitHub Desktop.

RS_CentralStandardStock8car.con

I checked the consist folder manually and it is missing.

Any help would be much appreciated.

Mike.

**Comment by DocMartin7644** [comment]
Hi Mike,

I checked my copy of the activity and it confirms that the consist is
indeed missing.  I have contacted the author of the activity, Richard
Stevenson, to let him know and ask him to correct the problem.  He will
either send me the consist or commit it to GitHub, at this point I don't
know which he will choose to do, but sooner or later this issue will be
sorted.

Regards,

Martin

**Reply by IsambardKingdomBrunel** [reply]
Thanks very much for replying Martin, much appreciated. No hurry regarding a fix, just thought i would mention it.

Regards,
Mike.

**Comment by DocMartin7644** [comment]
Hi Mike,
Richard has sent me the missing consist.  The simplest thing to do is for
you to send me your email and I will send you the consist as an
attachment.  My email is ***@***.***

Martin

**Comment by IsambardKingdomBrunel** [comment]
Hi Martin, that won't work buddy. ***@***.*** is not a valid email 
address LOL

Seriously though, my main email addy is ...................  
***@***.***

Many thanks for the help with the missing consist.

Regards,
Mike.


------ Original Message ------
From "DocMartin7644" ***@***.***>
To "DocMartin7644/Chiltern-Route-v2" 
***@***.***>
Cc "IsambardKingdomBrunel" ***@***.***>; "Author" 
***@***.***>
Date 01/07/2023 21:14:05
Subject Re: [DocMartin7644/Chiltern-Route-v2] Missing Consist RS 
Activity (Discussion #4)

>
>Hi Mike,
>Richard has sent me the missing consist. The simplest thing to do is 
>for
>you to send me your email and I will send you the consist as an
>attachment. My email is ***@***.***
>
>Martin
>
>On Fri, 30 Jun 2023 at 12:25, IsambardKingdomBrunel <

**Comment by DocMartin7644** [comment]
OK, Mike, GitHub is obviously hiding our email addresses, presumably for
our protection!  Let's try two things.  This is my email docmartin7644 AT
gmail.com - with AT of course standing for the @.

I am also attaching the consist to this message in the hope that GitHub
doesn't strip it off.

Let's see what works.

Martin

**Comment by greg2356** [comment]
An essential file is missing and Open Rails cannot continue.
C:\Trainsim\Chiltern-Route-v3\TRAINS CONSISTS\LJ_Railcar.CON
OK 

route 

An Evening at Princes Risborough the title is rs_An hour at Princes Risborough just watching trains go by no trains needed yet it states
Locomotive 4994 downtown hall early dont know why its missing the railcar when its just watching trains go by.


I been reading your activities Document files 

Golf Special works 

Rs Bakerloo Rush Hour error file not found consist LT_Faraday+5 Dreadnought.con yet it says LUL 1938 Tube Stock West

Banbury-Princes Risborough Auto works

Chiltern Court Coal Turn works

Football Special works

Great Central Inner Suburban error file not found consist LT_Faraday+5 Dreadnought.con yet it says RS_N2 + SetA Nbound

Milk to Marylebone works

Slipping at Bicester works

World Cup Shuttle works 

All other activitys i believe you must be working on

Thanks Greg email gregorywiederhold@yahoo.com

**Comment by DocMartin7644** [comment]
Hi Greg,

I attach the two consists that you are missing.  Let me know if you need
any of the stock that makes up these consists.

Martin

**Comment by greg2356** [comment]
Hi Martin,
I dont see the consists attached as files or link. I been reading up on how Open Rails work and also how developers like you make a route. I know it takes alot of time and effort  making a route to many folders and files. anyhow I had to wipe my laptop and format and reinstall everything every thing works alot better now. any way I been trying out your routes working or not i will let you know what files are missing usually a consist for trainset.
    On Wednesday, February 28, 2024 at 03:23:31 PM EST, DocMartin7644 ***@***.***> wrote:  
 
 

Hi Greg,

I attach the two consists that you are missing. Let me know if you need
any of the stock that makes up these consists.

Martin

**Comment by DocMartin7644** [comment]
Greg,
I don't know what went wrong but here are the two consists again.
[2Consists.zip](https://github.com/DocMartin7644/Chiltern-Route-v3/files/14454747/2Consists.zip)

Martin

**Comment by greg2356** [comment]
Hi Martin,
I installed the consists into the folders but stll have problems with theses routes.

RS_GREAT CENTRAL INNER SUNBURBAN ACTIVITY MISSING CoN FILE. TRAINS\CONSISTS\RS_D319_LE.CON I used that con file but its saying i need this one now? it says RS_N2 + Set 35A Nbound for consist
RS_BAKERLOO RUSH HOUR. missing file  LT_FARADAY +5DREADNOUGHTS.CON I put this file in the consist folder but it still says its missing?
rs_slipping at Bicester car derailed 0-13 at the beginning train is already traveling down the track at 35 miles per hour so i slow it down but I still get the warning.
I been going to trainsim.com and learning how routes are run and made hopefully I can figure out whats wrong with your consists 
    On Thursday, February 29, 2024 at 07:05:31 PM EST, DocMartin7644 ***@***.***> wrote:  
 
 


Greg,
I don't know what went wrong but here are the two consists again.
2Consists.zip

Martin

**Reply by DocMartin7644** [reply]
I attach 3 consists: (1) RS_N2+Set35A Nbound  (2) LT_Faraday+5Dreadnought.con (there was a small error in the name of the consist in the version I first set you) (3) RS_D3219_LE.con (I think this is probably the correct consist that you called RS_D319_LE.con)

I do not know why you get the derailment in the Slipping at Bicester activity.  I have run this activity several times without problem as has the author, Richard Stevenson.  Perhaps you need to check your Adhesion settings in the OR main menu Options > Simulation and Options > Experimental but I am just guessing as I have never needed to change these settings myself.
[Consists.zip](https://github.com/DocMartin7644/Chiltern-Route-v3/files/14474593/Consists.zip)

**Comment by greg2356** [comment]
thanks for the reply I might check the settings like you said I ran Slipping at Bicester a few weeks ago and that car would show derailed warning ran the route for an hour then another car saying its derailing then train just stopped on me. Tryied the route after reinstalling windows and programs ran the route and that car showed deraing again so i stopped the route. just looked at that setting its 100% default is 130% hah I run that route now
    On Sunday, March 3, 2024 at 04:11:48 PM EST, DocMartin7644 ***@***.***> wrote:  
 
 


I attach 3 consists: (1) RS_N2+Set35A Nbound (2) LT_Faraday+5Dreadnought.con (there was a small error in the name of the consist in the version I first set you) (3) RS_D3219_LE.con (I think this is probably the correct consist that you called RS_D319_LE.con)

I do not know why you get the derailment in the Slipping at Bicester activity. I have run this activity several times without problem as has the author, Richard Stevenson. Perhaps you need to check your Adhesion settings in the OR main menu Options > Simulation and Options > Experimental but I am just guessing as I have never needed to change these settings myself.
Consists.zip

**Comment by IsambardKingdomBrunel** [comment]
What has happened to the zip download option please Martin. I find all this github app etc very confusing, just would like to download the route. As i haven't had OR installed since last year.

**Comment by DocMartin7644** [comment]
Hi IKB,

For some reason the zip download option in the route description on GitHub has never worked and I do not know how to correct it.  However, if you click on the green "Code" button, it gives you the option to download the zip file and this does work

If you have installed GitHub Desktop, this allows you to update the route with the latest versions each time I add something to the route.  This is very useful since the route is still a work-in-progress and so I am continually updating it.

Good luck!

Martin

**Comment by IsambardKingdomBrunel** [comment]
Hi Martin,

thanks very much for replying and for the information.

No i cannot get on with the Github app sorry, so not installed buddy.

Thank you again.

Regards,
Mike.


------ Original Message ------
From "DocMartin7644" ***@***.******@***.***>>
To "DocMartin7644/Chiltern-Route-v3" ***@***.******@***.***>>
Cc "IsambardKingdomBrunel" ***@***.******@***.***>>; "Author" ***@***.******@***.***>>
Date 03/08/2024 15:05:58
Subject Re: [DocMartin7644/Chiltern-Route-v3] Missing Consist RS Activity (Discussion #4)


Hi IKB,

For some reason the zip download option in the route description on GitHub has never worked and I do not know how to correct it. However, if you click on the green "Code" button, it gives you the option to download the zip file and this does work

If you have installed GitHub Desktop, this allows you to update the route with the latest versions each time I add something to the route. This is very useful since the route is still a work-in-progress and so I am continually updating it.

Good luck!

Martin

**Reply by cjakeman** [reply]
Hi MIke,

> No i cannot get on with the Github app sorry, so not installed buddy. 

There is another option available if you are willing to switch to the Unstable Version of Open Rails.

This version, which is still experimental, has a Content button that leads to a page listing 6 routes that Open Rails can download and install for you. It using downloading code designed to work with Git and is suitable for models too large for downloading as a Zip archive.

![2024-08-04 19_07_36-Missing Consist RS Activity · DocMartin7644_Chiltern-Route-v3 · Discussion #4 — ](https://github.com/user-attachments/assets/81cae0ba-034a-4340-a8ed-38741d52864f)



Hope that helps,

Chris

**Comment by IsambardKingdomBrunel** [comment]
Hi Chris,

thanks very much for the input. I use the Monogame version of OR, which i believe is based on the unstable releases.

Your help is much appreciated buddy.

Regards,
Mike.


------ Original Message ------
From "Chris Jakeman" ***@***.******@***.***>>
To "DocMartin7644/Chiltern-Route-v3" ***@***.******@***.***>>
Cc "IsambardKingdomBrunel" ***@***.******@***.***>>; "Author" ***@***.******@***.***>>
Date 04/08/2024 19:12:49
Subject Re: [DocMartin7644/Chiltern-Route-v3] Missing Consist RS Activity (Discussion #4)


Hi MIke,

No i cannot get on with the Github app sorry, so not installed buddy.

There is another option available if you are willing to switch to the Unstable Version of Open Rails.

This version, which is still experimental, has a Content button that leads to a page listing 6 routes that Open Rails can download and install for you. It using downloading code designed to work with Git and is suitable for models too large for downloading as a Zip archive.

2024-08-04.19_07_36-Missing.Consist.RS.Activity.DocMartin7644_Chiltern-Route-v3.Discussion.4.jpg (view on web)<https://github.com/user-attachments/assets/81cae0ba-034a-4340-a8ed-38741d52864f>

Hope that helps,

Chris
